### PR TITLE
Graphic for README

### PR DESCRIPTION
**Pull request by joshuayoes**
## Please verify the following:

- [x] Everything works on iOS/Android
- [x] `yarn ci:test` **jest** tests pass with new tests, if relevant
- [x] `./docs/` has been updated with your changes, if relevant

## Describe your PR
Adding graphic for Jon Ruddell quote requested in this issue: [https://github.com/infinitered/ignite/issues/1467](https://github.com/infinitered/ignite/issues/1467)

Ignite Testimonial

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +25,7 @@
 
 And you also get (by default) all of the sweet, sweet goodness of [our default boilerplate, Bowser](https://github.com/infinitered/ignite-bowser), or choose [one of many others](./BOILERPLATES.md).
 
-> Ignite saved me a few weeks (if not months) getting started with React Native.
-
-_Jon Ruddell_
+<img width="946px" alt="Ignite saved me a few weeks (if not months) getting started with React Native - Jon Ruddel. Cofounder at SportsBooth. JHipster Core Team." src="https://i.imgur.com/etRUK0X.jpg">
```
Comment (ruddell) [inline README.md]:
```suggestion
<img width="946px" alt="Ignite saved me a few weeks (if not months) getting started with React Native - Jon Ruddell. Cofounder at SportsBooth. JHipster Core Team." src="https://i.imgur.com/etRUK0X.jpg">
```

Comment (ruddell) [reply README.md]: Nice image!

Comment (joshuayoes) [reply README.md]: Thanks! I'm happy it turned out well

**Comment by jamonholmgren** [comment]
@joshuayoes This is great, thank you!

**Pull request merged**